recipes/mongos.rb

```diff
@@ -47,6 +47,6 @@
 end
 
 mongodb_instance node['mongodb']['instance_name']['mongos'] do
-  mongodb_type 'mongos'
+  is_mongos true
```

Comment:
Would specifying `node['mongodb']['is_mongos']` be preferred here?  It is being forced to `true` at line 24 of this file, but that could be dropped if it isn't useful anymore

Reply:
I'm trying to move away from node attributes as much as possible, but you make a good point.

The reason I'm not pulling from `node['mongodb']['is_mongos']` is because this recipe is for `mongos`. It seems redundant to have `node.override['mongodb']['is_mongos'] = true` and then later pull from that attribute.